### DISCUSSION TITLE
Vite Multi-Page App

### DISCUSSION THREAD

**Discussion by f3oall** [posva/unplugin-vue-router#305] [Questions and Help]
Hey guys,

First of all - file-based routing is a great idea, it makes so much sense. Thank you for this lib.

My project has a client and admin area, so I'd like to use [Vite Multi-Page mode](https://vitejs.dev/guide/build#multi-page-app). 

How should I approach the integration of unplugin-vue-router in this case? Are there any caveats?

**Comment by posva** [comment]
It's not supported yet. https://github.com/posva/unplugin-vue-router/issues/148

**Reply by f3oall** [reply]
I see, is there any roadmap or date for this feature? Maybe I can help with it?

**Reply by posva** [reply]
There isn't anything planned yet. You can try to play with it with a fork. Note that the current version being worked is the branch from https://github.com/posva/unplugin-vue-router/pull/246. So it might be a bit difficult to work on it. That being said, you could always experiment locally and propose API ideas and such. To be transparent, multi-page support is behind a lot of other features on my list, so I don't plan giving it a look until May-July. Which means there is plenty of time to test out ideas and API designs to bootstrap it faster when the time comes!

**Reply by f3oall** [reply]
Sounds good, I'll go with monorepo for now, and when I have some spare time I'll look into the issue. Should be earlier than May-July :) Is there any way to find current active branch, or should I ping you about it then?

**Reply by posva** [reply]
It's the one I sent you until it gets merged into main

**Comment by posva** [comment]
Tracking at #148
